.github/workflows/ci.yml

```diff
@@ -0,0 +1,69 @@
+name: ci
+
+on: [push]
+
+jobs:
+  phpunit:
+    if: "!contains(github.event.head_commit.message, 'skip ci')"
```

Comment:
For what use cases is this line here?

Reply:
`CI` can be skipped when the commint message contains "skip ci"

Reply:
github has been resolved https://github.blog/changelog/2021-02-08-github-actions-skip-pull-request-and-push-workflows-with-skip-ci